src/AccordionSection/AccordionSection.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import { heightOfAccordionHeader } from "../Accordion/Accordion"
+import Icon from "../Icon/Icon"
+import { DefaultProps } from "../types"
+import { useUniqueId } from "../useUniqueId"
+import styled from "../utils/styled"
+
+export interface AccordionSectionProps extends DefaultProps {
+  title: React.ReactNode
+  children: React.ReactNode
+  id?: string
+  tabindex?: number
+  expanded?: boolean
+  toggleExpanded?: () => void
+}
+
+const Container = styled("div")<{ expanded: boolean }>`
+  label: AccordionSection;
+  overflow: hidden;
+`
+
+const Header = styled("div", { shouldForwardProp: () => true })<{ tabindex: number }>`
```

Comment:
```suggestion
const Header = styled("div")<{ tabIndex: number }>`
```

I think all props are forwarded by default.